### PR TITLE
House 1

### PR DESCRIPTION
**Pull request by SmartCode18**
Have an issue with ChromeDriver.  Probably dependency issue but not able to figure out. Will focus on it later on.

### CODE REVIEW

File: src/main/java/tests/RepresentativesPageValidation.java [hunk 1]
```diff
@@ -1,4 +1,22 @@
 package tests;
 
+import domain.pages.RepresentativesPage;
+import org.junit.Test;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.support.PageFactory;
+
 public class RepresentativesPageValidation {
```
Comment (fairwinds0099) [inline src/main/java/tests/RepresentativesPageValidation.java]: rename class RepPageValidationTests

File: src/main/java/tests/RepresentativesPageValidation.java [hunk 2]
```diff
@@ -1,4 +1,22 @@
...
+
+    @Test
+    public void verifyRepresentativesPageLogo(){
```
Comment (fairwinds0099) [inline src/main/java/tests/RepresentativesPageValidation.java]: rename method repPageLogoShouldAppear

File: src/main/java/tests/RepresentativesPageValidation.java [hunk 3]
```diff
@@ -1,4 +1,22 @@
...
+        System.setProperty("WebDriver.chrome.driver", "resources/chromedriver");
+        WebDriver driver = new ChromeDriver();
+        RepresentativesPage representativesPage = new PageFactory().initElements(driver, RepresentativesPage.class);
+
+        driver.get("https://www.house.gov");
+        driver.findElement(By.linkText("Representatives")).click();
```
Comment (fairwinds0099) [inline src/main/java/tests/RepresentativesPageValidation.java]: create this click method in domain (Homepage) and call it here using a homePage object

File: src/main/java/tests/RepresentativesPageValidation.java [hunk 4]
```diff
@@ -1,4 +1,22 @@
...
+        System.out.println(RepresentativesPage.isRepresentativesPageStateNameDisplayed());
```
Comment (fairwinds0099) [inline src/main/java/tests/RepresentativesPageValidation.java]: finish with Assertions

File: src/main/java/tests/HomePageValidation.java [hunk 5]
```diff
@@ -6,15 +6,18 @@
...
 
     @Test
     public void verifyHomePageLogo(){
         System.setProperty("webdriver.chrome.driver", "resources/chromedriver");
         WebDriver driver = new ChromeDriver();
         HomePage homePage = new PageFactory().initElements(driver, HomePage.class);
+
         driver.get("https://www.house.gov");
-        System.out.println(homePage.isHomePageLogoPresent());
+        System.out.println(homePage.isHomePageZipCodeInputPresent());
```
Comment (fairwinds0099) [inline src/main/java/tests/HomePageValidation.java]: finish with Assertions

File: src/main/java/domain/pages/HomePage.java [hunk 6]
```diff
@@ -1,4 +1,29 @@
...
+        private WebElement homePageZipCodeInput;
+
+        @FindBy (xpath = "(//a[href='/representatives'])[1]")
+        private WebElement representativesLink;
+
+        public HomePage(WebDriver driver) {
+            this.driver = driver;
+        }
+
+        public boolean isHomePageZipCodeInputPresent(){
```
Comment (fairwinds0099) [inline src/main/java/domain/pages/HomePage.java]: good

File: pom.xml [hunk 7]
```diff
@@ -103,6 +101,36 @@
             <artifactId>rest-assured</artifactId>
             <version>3.1.1</version>
         </dependency>
+        <dependency>
+            <groupId>org.seleniumhq.selenium</groupId>
+            <artifactId>selenium-support</artifactId>
+            <version>2.0a7</version>
```
Comment (fairwinds0099) [inline pom.xml]: is this version correct?

Comment (SmartCode18) [reply pom.xml]: Updated to 3.12.0

File: src/main/java/tests/HomePageValidation.java (hunk 5 above)
Comment (fairwinds0099) [reply src/main/java/tests/HomePageValidation.java]: Assert.assertTrue(isHomePageZipCodeInputPresent);

**Comment by SmartCode18** [comment]
All Three Test Cases are working fine

File: src/main/java/tests/RepPageValidationTests.java [hunk 8]
```diff
@@ -0,0 +1,27 @@
...
+            System.setProperty("webdriver.chrome.driver", "resources/chrome-driver");
+            WebDriver driver = new ChromeDriver();
+            HomePage homePage = new PageFactory().initElements(driver, HomePage.class);
+            RepresentativesPage representativesPage = new PageFactory().initElements(driver, RepresentativesPage.class);
+
+
+            driver.get("https://www.house.gov");
+            homePage.isHomePageZipCodeInputPresent();
+            homePage.clickRepresentativesLink();
+            System.out.println(representativesPage.isRepresentativesPageStateNameDisplayed());
```
Comment (fairwinds0099) [inline src/main/java/tests/RepPageValidationTests.java]: add Assertion here

**Pull request closed without merging**